conda_build/render.py

```diff
@@ -2,36 +2,43 @@
 # SPDX-License-Identifier: BSD-3-Clause
```

Comment:
we might be able to pull these changes into a separate PR or rebase with the reformat